packages/adapter-d1/src/d1.ts

```diff
@@ -12,6 +12,7 @@ import {
   TransactionOptions,
 } from '@prisma/driver-adapter-utils'
 
+import { tags } from '../../internals/src/logger'
```

Comment:
Note: this breaks in CI with `Error: s/adapter-d1 build: ✘ [ERROR] JSON validation failed:`
See https://github.com/prisma/prisma/actions/runs/7699595068/job/20981580332?pr=22853#step:3:241